servicetalk-opentelemetry/src/main/java/io/servicetalk/opentelemetry/OpenTelemetryHttpServerFilter.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright © 2022 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicetalk.opentelemetry;
```

Comment:
We usually try to highlight in the module name which protocol it targets. Consider splitting it into two modules:
- `servicetalk-opentelemetry-asynccontext`
- `servicetalk-opentelemetry-http`

Reply:
I created two modules as suggested.